lib/fixtures/set-triggers.sql

```diff
@@ -0,0 +1,24 @@
+-- Delete triggers
+DROP TRIGGER IF EXISTS toggleOrgUsers on organisations;
+
+-- Create procedures
+CREATE OR REPLACE FUNCTION toggleOrgUsers()
+RETURNS TRIGGER AS
+$BODY$
+  BEGIN
+    UPDATE people
+    SET active = NOT people.active
+    WHERE org_id = OLD.id;
+
+    RETURN NEW;
+  END;
+$BODY$
+LANGUAGE plpgsql;
+
+-- Create triggers
+CREATE TRIGGER toggleOrgUsers
+AFTER
+UPDATE ON organisations
+FOR EACH ROW
+WHEN (OLD.active IS DISTINCT FROM NEW.active)
+EXECUTE PROCEDURE toggleOrgUsers();
```

Comment:
From the sql function, it looks like an inactive primary user would become activated when we toggle the linked organisations activity status.. 

A quick work-around could be to call this function only when the org is being deactivated. Or perhaps we can pass the function a parameter which indicates if the org is being activated or deactivated?